musicbot/scrobble.py

```diff
@@ -0,0 +1,78 @@
+import urllib
+import json
+import hashlib
+import time
+import sys
+
+sk = ''
+api_key = ''
+api_secret = ''
+
+class Scrobble(object):
+	def __init__(self, user, passwd, key, secret):
+		global sk
+		global api_key
+		global api_secret
```

Comment:
Literally my first experience with python. Still learning. I appreciate the quick response.